tools/compiletest/src/read2.rs

```diff
@@ -6,77 +6,29 @@
 // FIXME: This is a complete copy of `cargo/src/cargo/util/read2.rs`
 // Consider unify the read2() in libstd, cargo and this to prevent further code duplication.
 
-pub use self::imp::read2;
+// Note: read2_abbreviated was replaced by read2 because the original function skipped some of the output,
+// which caused issues with our tests (in particular tests with large outputs).
```

Comment:
Appreciate adding this comment, but if you remove `read2_abbreviated` it's surely going to confuse whoever works on this file in the future. Three ways to approach this:
 * Remove `read2_abbreviated` and don't add the comment.
 * Keep `read2_abbreviated` and add the comment for comparison.
 * Remove `read2_abbreviated` and add a comment in new line 90 in `tools/compiletest/src/runtest.rs` that refers to it (explaining it shortened the output) and explain that this is not desirable behavior, so you just the full output instead.

Reply:
I decided to remove `read2_abbreviated` and the comment.